Agents/APIAgent/tbox_dev/rmlts.owl

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0"?>
+<rdf:RDF xmlns="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#"
+     xml:base="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     >
+    <owl:Ontology rdf:about="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl"/>
+
+    <!--
+    ///////////////////////////////////////////////////////////////////////////////////////
+    //
+    // Datatype Properties
+    //
+    ///////////////////////////////////////////////////////////////////////////////////////
+     -->
+    <owl:DatatypeProperty rdf:about="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#time_iterator">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#TimeSeriesMap"/>
+    </owl:DatatypeProperty>
+
+
+    <owl:DatatypeProperty rdf:about="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#value_iterator">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/ontology/rml4ts/rml4ts.owl#TimeSeriesMap"/>
+    </owl:DatatypeProperty>
```

Comment:
I have modified it. However since this file is going to the JPS_ONTOLOGY folder, I suppose I should delete it in this repository and later commit to the JPS_ONTOLOGY in a separate branch.

Reply:
As commented, I believe it would be preferable to simply move the file as part of this pull request.

Reply:
I have checked the TBOX generator, however, I have questions concerning that it does not seem to contain a way to define NamedIndividual. Does it mean we are not using this definition anymore? Either way, I put an manually edited updated version tentatively in the latest pull request.

Reply:
The current design of the TBoxGenerator indeed only supports TBox statements, i.e. concepts/properties, but not instances, i.e. named individuals.

Since the ontology is tiny, it makes most sense to take no further action in this case.